integration_tests/dbt_project.yml

```diff
@@ -59,6 +59,8 @@ seeds:
 
 vars:
   # ensure integration tests run successfully when there are 0 of a given model type (extra)
-  model_types: ['staging', 'intermediate', 'marts', 'other', 'extra']
+  model_types: ['staging', 'intermediate', 'marts', 'other', 'extra', 'new_model_type']
   # dummy variable used for testing fct_hard_coded_references
-  my_table_reference: 'grace_table'
\ No newline at end of file
+  my_table_reference: 'grace_table'
+  new_model_type_folder_name: 'my_new_models'
+  new_model_type_prefixes: 'nwmdl_'
```

Comment:
Should we check that a prefix with _ works to? Like `nw_mods_`

Reply:
I don't think we do anything to the value of these variables anywhere, we only do string manipulation on the name of the variable after we loop. 

One other thing we could do is refactor `loop_vars` to contain the suffix-appending logic, and then we might be able to remove the string parsing thing altogether